### PR TITLE
Create PERMISSIONS.md

### PR DESCRIPTION
**Pull request by PhilipFlyvholm**
Implemented #51

### CODE REVIEW

**Comment by PhilipFlyvholm** [comment]
@PatrickMatthiesen When you have time, no pressure on this PR, can you check if this aligns with what we have written in the chrome web store reviews? 🎉

File: PERMISSIONS.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+### Scripting
+This is the most important permission since it allows us to inject scripts and CSS into the page.
+This is used in the following ways:
+- Injecting the selected theme CSS, such that you don't need to load all the themes on every load, but only the one you have selected.
+- Injecting a script to modify the root by adding a `dark` class, such that the dark mode is shown directly when the CSS is loaded and not after the page load.
+
+### webNavigation
+WebNavigation is used to be the first code to know that you are about to enter LearnIT. This sounds a bit weird, however, this is needed to ensure that the page does not flash white when loading in dark mode.
+Some browsers may say that we access your browser history because of this. This is only since we inject the scripts at the moment between page switching and we could in theory use this to get access to your browser history.
```
[the remark this answers is not in the document]
Comment (PhilipFlyvholm) [reply PERMISSIONS.md:29]: Hmm do you mean like mentioning the [event order](https://developer.chrome.com/docs/extensions/reference/webNavigation/#event-order) and so on or which functions that can be used to track the history?

[the remark this answers is not in the document]
Comment (PatrickMatthiesen) [reply PERMISSIONS.md:29]: kinda. I was thinking that if we add we need to inject something at a time where we theoretically have access to the URL, then Google says we have access to the history as we can save the URL if we want, but that we don't do that as we don't need it for anything. 
Btw we can back it up if we still have that security thing that blocks most network traffic for the extension.

**Review by PatrickMatthiesen — APPROVED** [review]
looks good :)

**Pull request merged**